.github/workflows/behaviour.yml

```
@@ -1,5 +1,16 @@
 name: Behaviour tests
-on: [ push, pull_request ]
+on:
+  push:
+    # Do not run after merging a pull request
+    branches-ignore:
+      - 'develop'
+      - '8.0.x'
+      - '1.7.8.x'
```

```suggestion
      - '**.x'
```
With a wildcard we would anticipate the future branches as well no? Like 8.1.x, 9.0.x, ...

---

Other alternative
```suggestion
      - '*.*.x'
      - '1.7.8.x'
```

---

First suggestion is better to take out 1.7.8.x too